src/test/java/com/stripe/model/AccountCreateParamsTest.java

```diff
@@ -0,0 +1,90 @@
+package com.stripe.model;
+
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertNotNull;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.IntStream;
+import org.junit.Test;
+
+public class AccountCreateParamsTest {
+  @Test
+  public void testBuilderCreateParams() {
+    AccountCreateParams.Builder builder = AccountCreateParams.builder();
+    AccountCreateParams createParams = builder
+        .beginLegalEntity()
+          .beginArrayAdditionalOwners()
+            .beginAdditionalOwner()
+              .beginAddress()
+                .setCity("Tokyo")
+                .setCountry("Japan")
+                .setPostalCode("123")
+              .done()
+            .done()
+          .beginAdditionalOwner()
+            .beginAddress()
+              .setCity("London")
+              .setCountry("UK")
+              .setPostalCode("456")
+            .done()
+          .done()
+        .done()
+      .done()
+      .setType("foo_type")
+    .build();
+
+    AccountCreateParams.LegalEntity legalEntity = createParams.getLegalEntity();
+    assertNotNull(legalEntity);
+    assertNotNull(legalEntity.getAdditionalOwners());
+    assertEquals(legalEntity.getAdditionalOwners().size(), 2);
+    AccountCreateParams.Address address1 = legalEntity.getAdditionalOwners().get(0).getAddress();
+    assertEquals("Tokyo", address1.getCity());
+    assertEquals("Japan", address1.getCountry());
+    assertEquals("123", address1.getPostalCode());
+
+    AccountCreateParams.Address address2 = legalEntity.getAdditionalOwners().get(1).getAddress();
+    assertEquals("London", address2.getCity());
+    assertEquals("UK", address2.getCountry());
+    assertEquals("456", address2.getPostalCode());
+
+    assertEquals("foo_type", createParams.getType());
+  }
+
+  @Test
+  public void testBuilderCreateParamsWithIterations() {
```

Comment:
Users can still programmatically manipulate a subgraph of this builder pattern.
However this nested builder pattern makes it hard to build sub-hash independently in different scope.